internal/db/utils.go

```diff
@@ -45,3 +47,33 @@ func NeedsEscaping(name string) bool {
 	}
 	return false
 }
+
+var explainQueryPlanStatement = "EXPLAIN QUERY PLAN"
+var explainQueryPlanColumnNames = []string{"id", "parent", "notused", "detail"}
+
+func queryContainsExplainQueryPlanStatement(query string) bool {
+	return strings.Contains(
```

Comment:
Maybe a `StartsWith` is safer now, what do you think?

Reply:
According to the docs: https://www.sqlite.org/lang_explain.html
`EXPLAIN QUERY PLAN` always appears in the beggining of statements, bug I agree that having some extra safety woud be nice.